packet/templates/packet.html

```diff
@@ -135,26 +134,24 @@ <h5>Upperclassmen Score - {{ '%0.2f' % upperclassmen_percent }}%</h5>
                 <div class="card mb-2">
                     <div class="card-header">
                         <b>On-Floor Freshmen Signatures</b>
-                        <b class="signature-count">{{ signed_dict.freshmen }}/{{ signatures.freshmen | count }}</b>
+                        <b class="signature-count">{{ received.fresh }}/{{ required.fresh }}</b>
                     </div>
                     <div class="card-body table-fill">
                         <div class="table-responsive">
                             <table class="table table-striped no-bottom-margin" data-module="table"
                                    data-searchable="true" data-sort-column="3" data-sort-order="asc"
                                    data-length-changable="true" data-paginated="false">
                                 <tbody>
-                                {% for m in signatures.freshmen %}
-                                    <tr {% if m.signed %}style="background-color: #4caf505e" {% endif %}>
+                                {% for sig in packet.fresh_signatures %}
+                                    <tr {% if sig.signed %}style="background-color: #4caf505e" {% endif %}>
                                         <td>
-                                            <a href="/packet/{{ m.freshman_username }}">
-                                                <img class="eval-user-img" alt="{{ m.freshman_username }}"
-                                                     src="https://www.gravatar.com/avatar/freshmen?d=mp&f=y"
-                                                     width="25"
-                                                     height="25"/> {{ m.name }} ({{ m.freshman_username }})
-                                            </a>
+                                            <img class="eval-user-img" alt="{{ sig.freshman_username }}"
+                                                 src="https://www.gravatar.com/avatar/freshmen?d=mp&f=y"
```

Comment:
It'd be nice to attempt to pull a Gravatar based on their RIT email, but I'm not sure about the performance implications of MD5ing all of their RIT emails.

Reply:
That's an interesting idea. We'll have to look into that when rebuilding the UI.